packages/devtools/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@react-navigation/devtools",
+  "description": "Developer tools for React Navigation",
+  "version": "5.0.0",
+  "keywords": [
+    "react",
+    "react-native",
+    "react-navigation"
+  ],
+  "license": "MIT",
+  "repository": "https://github.com/react-navigation/react-navigation/tree/master/packages/devtools",
+  "bugs": {
+    "url": "https://github.com/react-navigation/react-navigation/issues"
+  },
+  "homepage": "https://reactnavigation.org",
+  "main": "lib/commonjs/index.js",
+  "react-native": "src/index.tsx",
+  "source": "src/index.tsx",
+  "module": "lib/module/index.js",
+  "types": "lib/typescript/src/index.d.ts",
+  "files": [
+    "src",
+    "lib",
+    "!**/__tests__"
+  ],
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "prepare": "bob build",
+    "clean": "del lib"
+  },
+  "dependencies": {
+    "@react-navigation/core": "^5.10.0",
```

Comment:
I guess you considered `peerDependencies` and it probably didn't work given how it's transitive through `native` in most cases... a little worried about two versions of `@react-navigation/core` getting installed if somebody forgets to update `@react-navigation/devtools` when upgrading other packages, but can't think of a better solution

Reply:
We just import types from this package so it would be fine.